types/k6/test/timers.ts

```diff
@@ -33,7 +33,16 @@ setInterval('functionRef', 5);
 // @ts-expect-error
 setInterval('functionRef', 5, 'arg');
 
+// @ts-expect-no-error
```

Comment:
```suggestion
```

I don't think this exist as a syntax, when you want a none error you just leave it